src/FamilyHubs.Referral.Web/Pages/Shared/_TextBoxDetail.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model FamilyHubs.Referral.Core.Models.PartialTextBoxViewModel
+@if (Model.ValidationValid == false)
+{
+    <div class="govuk-error-summary" data-module="govuk-error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
+            There is a problem
+        </h2>
+        <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+                <li>
+                    <a href="#consent" data-testid="enter-service-name-error">@Model.MainErrorText</a>
```

Comment:
The id in the href needs to be a fragment with the id of the failing input control, so that the link works.

Reply:
Changed to #TextBoxValue